### ISSUE TITLE
Drupal 8 has /vendor folder and Buildpack fails

### ISSUE THREAD

**Issue by devekko** [fgrehm/devstep#106]
I managed to get Drupal 7, Wordpress 4, Joomla 3.45 to installation stage on Devstep

BUT Drupal 8 fails, its got a vendor folder and the Devstep PHP Buildpack logic fails an already existing vendor folder

niccolox@trustyinx:~/Projects/devstep/drupal-8-core|8.0.x⚡
⇒  devstep hack -p 8075:8075
==> Creating container using 'fgrehm/devstep:v0.4.0'
-----> Building project at '/workspace' with the following buildpacks:
       - /opt/devstep/buildpacks/php
-----> Building with /opt/devstep/buildpacks/php
       /workspace/vendor cant be a directory

is there a way around this?

can I use alternative Buildpacks in Devstep?

I was looking at the list of Buildpacks in Tsuru and Herokuish and did some simple compares

**Comment by fgrehm** [comment]
> is there a way around this?

Not from the top of my head, sorry. I'm planning to dedicate some time for devstep at some point this month and will look into this before giving you a final answer.

> can I use alternative Buildpacks in Devstep?

Sure, see http://fgrehm.viewdocs.io/devstep/buildpacks/custom/ for more :beers:

**Comment by devekko** [comment]
awesome

was wondering how devstep will deal with two PHP buildpacks

can I turn the devstep default internal one off?

**Comment by fgrehm** [comment]
You can mount the buildpack on top of devstep's default one :smirk:

**Comment by devekko** [comment]
ok, will try this in coming weeks...

**Comment by devekko** [comment]
##THANKS

**Comment by fgrehm** [comment]
Now I remember, is Drupal's `vendor` dir the place where composer's dependencies are kept?

**Comment by devekko** [comment]
http://cgit.drupalcode.org/drupal/tree/vendor?h=8.1.x

**Comment by fgrehm** [comment]
looks like it is, lemme see if I can squeeze in a fix for it for 1.0

**Comment by devekko** [comment]
yes, I think you are correct to assume that, but its Drupal 8 and I am not exactly sure how they work with composer, the composer.json doesnt reference those libs so, maybe they are placed into the tree some other Drupal-y way
